From f256ecb8d02a77bfd5c21285329c11520c4a407c Mon Sep 17 00:00:00 2001
Message-Id: <f256ecb8d02a77bfd5c21285329c11520c4a407c.1539506014.git.devel@agrez.net>
From: Vaughan Agrez <devel@agrez.net>
Date: Sun, 14 Oct 2018 16:33:18 +0800
Subject: [PATCH] Update Makefile.ffmpeg

---
 Makefile.ffmpeg | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/Makefile.ffmpeg b/Makefile.ffmpeg
index 0ed4419..8f30f9b 100644
--- a/Makefile.ffmpeg
+++ b/Makefile.ffmpeg
@@ -20,11 +20,13 @@ configure:
 	CFLAGS="$(CFLAGS) ${INCLUDES}" \
 	LDFLAGS="" \
   ./configure \
-			--extra-cflags="-mfpu=vfp -mfloat-abi=hard -mno-apcs-stack-check -mstructure-size-boundary=32 -mno-sched-prolog" \
+			--extra-cflags="-mfpu=neon-vfpv4 -mfloat-abi=hard -mno-apcs-stack-check -mstructure-size-boundary=32 -mno-sched-prolog" \
 			--enable-shared \
 			--disable-static \
 			--arch=arm \
-			--cpu=arm1176jzf-s \
+			--cpu=armv7-a \
+			--enable-vfpv3 \
+			--enable-thumb \
 			--target-os=linux \
 			--disable-hwaccels \
 			--enable-parsers \
@@ -46,9 +48,7 @@ configure:
 			--enable-pthreads \
 			--enable-pic \
 			--disable-armv5te \
-			--disable-neon \
-			--enable-armv6t2 \
-			--enable-armv6 \
+			--enable-neon \
 			--enable-hardcoded-tables \
 			--disable-runtime-cpudetect \
 			--disable-debug \
-- 
2.17.1

