From f3afd122cf0fb235d5dc7cb7420399e7eea1c8d6 Mon Sep 17 00:00:00 2001
Message-Id: <f3afd122cf0fb235d5dc7cb7420399e7eea1c8d6.1500984020.git.devel@agrez.net>
From: Vaughan Agrez <devel@agrez.net>
Date: Tue, 25 Jul 2017 20:00:10 +0800
Subject: [PATCH] Fix ffmpeg configure when using openssl >= 1.1.0

---
 configure | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ffmpeg/configure b/ffmpeg/configure
index 592df19..54d8771 100755
--- a/ffmpeg/configure
+++ b/ffmpeg/configure
@@ -5788,7 +5788,7 @@ enabled omx               && { check_header OMX_Core.h ||
                                     add_cflags -isystem/opt/vc/include/IL ; }
                                 check_header OMX_Core.h ; } ||
                                die "ERROR: OpenMAX IL headers not found"; }
-enabled openssl           && { use_pkg_config openssl openssl/ssl.h SSL_library_init ||
+enabled openssl           && { use_pkg_config openssl openssl/ssl.h OPENSSL_init_ssl ||
                                check_lib openssl/ssl.h SSL_library_init -lssl -lcrypto ||
                                check_lib openssl/ssl.h SSL_library_init -lssl32 -leay32 ||
                                check_lib openssl/ssl.h SSL_library_init -lssl -lcrypto -lws2_32 -lgdi32 ||
-- 
2.9.4

